src/ensembl/src/store.ts

```diff
@@ -15,7 +15,10 @@ const googleAnalyticsMiddleWare = (store: any) => (next: any) => (
   if (action.meta && action.meta.ga && action.meta.ga.category) {
     // The action and category fields are mandatory
     GoogleAnalyticsTracking.trackEvent(action);
-  } else if (action.type === LOCATION_CHANGE) {
+  } else if (
+    action.type === LOCATION_CHANGE &&
+    action.payload.action !== 'REPLACE'
```

Comment:
no, we shouldn't filter out replaces; replaces are fine.

Reply:
Cool, I've updated it to check if the path has changed.